Rakefile

```diff
@@ -10,12 +10,36 @@ end
 
 task :default => :test
 
+# Because Bump ignores gemfiles
+module BumpAllGemfiles
+  def commit(version, file, options)
+    return unless File.directory?(".git")
+    system("rake bundle_all") if options[:bundle]
+    system("git add --update gemfiles") if options[:bundle]
+    super
+  end
+end
+Bump::Bump.singleton_class.prepend(BumpAllGemfiles)
+
 desc "Bundle all gemfiles"
 task :bundle_all do
   Bundler.with_original_env do
     system("which -s matching_bundle") || abort("gem install matching_bundle")
+    sh "BUNDLE_GEMFILE=Gemfile matching_bundle"
```

Comment:
does not work since it needs to be bundled to be able to execute the rake task

Reply:
I am assuming that someone will `bundle install` after cloning the repo.

Reply:
they have to bundle install to be able to run rake, so if they are in rake then they have bundled already

Reply:
Ah I see. I misunderstood. It's not that it doesn't work, it's that this line is not necessary.